apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -967,6 +967,31 @@ export const OrganizationWrapper: React.FC<
           <Box className="rs_read" paddingTop={fullWidthContent ? 0 : 4}>
             {mainContent ?? children}
           </Box>
+          {isMobile && (
+            <>
+              {(organizationPage.sidebarCards ?? []).map((card) => {
+                if (card.__typename === 'SidebarCard') {
+                  return (
+                    <Box marginY={3}>
```

Comment:
What if instead of adding a marginY=3 to the box we instead use a <Stack> around the map function call. That way we aren't pushing other elements away from this container